src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -92,23 +95,41 @@ class NonlinearSolverParameters : public dataRepository::Group
 
   /**
    * @brief Calculates the upper limit for the number of iterations to allow a
-   * cut to the next timestep.
-   * @return The scaled value of the limit (m_dtCutIterLimit * m_maxIterNewton)
+   * decrease to the next time step.
+   * @return The scaled value of the limit (m_timeStepDecreaseIterLimit * m_maxIterNewton)
    */
-  integer dtCutIterLimit() const
+  integer timeStepDecreaseIterLimit() const
```

Comment:
Maybe name it  `currentTimeStepDecreaseIterLimit` as it is not a direct getter but return the applied value ?

Reply:
Hi @jafranc, yes I see your point, however here the `timeStepDecreaseIterLimit` function returns a value that is constant for all the time steps of the simulations. So, do you mind if I keep this name to make it clear that the returned value is not specific to the current time step?

Reply:
Not at all. We can work with this name :)